Doc/library/collections.abc.rst

```diff
@@ -250,6 +253,10 @@ Collections Abstract Base Classes -- Detailed Descriptions
    :meth:`~generator.send`,
    :meth:`~generator.throw` and :meth:`~generator.close` methods.
 
+   Refer to
+   :ref:`annotating generators and coroutines <annotating-generators-and-coroutines>`
```

Comment:
I would probably just do the cross-references like this throughout:

```suggestion
   :ref:`annotating-generators-and-coroutines`
```

The only difference is that the word "annotating" is capitalized... but I think that's actually an improvement, as it makes it more obvious that it's the name of a section:

![image](https://github.com/user-attachments/assets/328e5f43-f697-401d-89d6-b7cda66bfb20)

Reply:
Yeah, agreed - I added this exactly to fight capitalization, but perhaps it's really better to retain it.